payments/stripe/__init__.py

```diff
@@ -1,5 +1,10 @@
 import json
```

Comment:
Can you remove this unused import?

I merged your changes into `main` locally and removed this import in the process, but this repo is set up with some strict rules so I can't push that local branch myself :sweat: